### ISSUE TITLE
Fails build because of contents in /build/temp/

### ISSUE THREAD

**Issue by danielbbronson** [ThrowTheSwitch/Ceedling#290]
I'm having an issue where ceedling will fail to build a second time because of the contents found in the /build/temp/ folder.

After the first time that the test is run (and the contents of /build/ are deleted), everything runs and passes as expected.

If I try to immediately run the test again, then I get compilation errors because of a copy of my test file found in the build/temp/ file that for whatever reason adds a bunch of '@@@@' signs before the includes.
![unnamed](https://user-images.githubusercontent.com/42743590/44665774-bd65c080-a9e4-11e8-88b3-021e24776311.jpg)

**Comment by laurensmiers** [comment]
I had a similar issue a few months ago.
In my project.yml file, I included the ceedling build folder as well (unintentionally but my glob/regex for including folders was too aggressive) and since the (temporary) header files in there are the same name as those included in the tests/sources, it will take those and try to compile your sources with these header files.
From your shell command, it looks like your including the temp build folder as well (-I ../test/build/temp).

**Comment by danielbbronson** [comment]
@laurensmiers, you are correct!  The temp compilation was being recursively added to the project.  Once I shifted around the project structure, everything works as expected.  Thank you!
